doc/man3/SSL_CTX_set_session_ticket_cb.pod

```diff
@@ -39,12 +39,13 @@ is the same as that given to SSL_CTX_set_session_ticket_cb(). The B<gen_cb>
 callback is defined as type B<SSL_CTX_generate_session_ticket_fn>.
 
 B<dec_cb> is the application defined callback invoked after session ticket
-decryption has been attempted and any session ticket application data is available.
-The application can call SSL_SESSION_get_ticket_appdata() at this time to retrieve
-the application data. The value of B<arg> is the same as that given to
-SSL_CTX_set_session_ticket_cb(). The B<retv> argument is the result of the ticket
-decryption. The B<keyname> and B<keyname_len> identify the key used to decrypt the
-session ticket. The B<dec_cb> callback is defined as type
+decryption has been attempted and any session ticket application data is
+available (if any). If ticket decryption was successful then the B<ss> argument
```

Comment:
The `(if any)` looks redundant given the `and any` on the previous line.

Reply:
It looks from the code like in TLS 1.3 multiple PSK identities might be sent, even multiple tickets, and we keep decrypting until one succeeds.  What does that mean for the contract with the callbacks?  Applications should not be too eager to force an abort if they don't like a (valid) ticket, because there may be another ticket? Is forcing an abort a good idea? They can always hang up after the handshake completes, and at that time they'll have seen all the tickets...  This is tricky I think.

Reply:
I removed the redundant "if any". Also added a caution in the `SSL_TICKET_RETURN_ABORT` section about multi-tickets.